parser/v2/parser.go

```diff
@@ -179,6 +183,18 @@ func (p *Parser) ParseEthLogs(_ context.Context, eventsData types.EventsData) (*
 	return &types.EventsParsedResult{EVMEvents: len(parsed), ParsedEvents: parsed}, nil
 }
 
+func (p *Parser) ParseMultisigEvents(ctx context.Context, txs []*types.Transaction, tipsetCid string, tipsetKey filTypes.TipSetKey) (*types.MultisigEvents, error) {
+	multisigTxs, err := p.helper.FilterTxsByActorType(ctx, txs, manifest.MultisigKey, tipsetKey)
```

Comment:
I would filter by actor in a previous step before this func call, just in case in the future you might want to do another kind of processing to msig txs and thus avoiding having to filter again.